tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -197,19 +204,20 @@ def test_extract_documentation_PYTHON3(self):
         expected = list(DocumentationComment(r.documentation,
                                              r.language,
                                              "doxygen",
+                                             r.indent,
                                              r.marker,
                                              r.range)
                         for r in expected)
 
-        expected.insert(4, DocumentationComment(
+        expected.insert(5, DocumentationComment(
```

Comment:
ah yeah nvm, I somehow thought this is the column indentation, don't know from where my brain has taken this up :3